starboard/android/shared/toolchain/toolchain.gni

```diff
@@ -21,8 +21,10 @@ declare_args() {
   android_ndk_api_level = 24
 }
 
+_home = getenv("HOME")
+
 if (android_sdk_path == "") {
-  android_sdk_path = "$starboard_toolchains_path/AndroidSdk"
+  android_sdk_path = "${_home}/starboard_toolchains/AndroidSdk"
```

Comment:
I'd leave a TODO to try and figure out how to manage this in more chrome-standard way later. Good for now

Reply:
It's actually not even used, just imported and has issues because the imported build file no longer defines starboard_toolchains_path—I can add a TODO to remove this all